Assignment 3.Rmd

```diff
@@ -46,12 +43,9 @@ K3 <- na.omit(K2) #This command create a data frame with only those people with
 Another pre-processing step used in K-means is to standardize the values so that they have the same range. We do this because we want to treat each week as equally important - if we do not standardise then the week with the largest range will have the greatest impact on which clusters are formed. We standardise the values by using the "scale()" command.
 
 ```{r}
-
 K3 <- scale(K3)
-
 ```
 
```

Comment:
It may be worth making a note for yourself that explains why you scaled the data so that if you need to do this analysis again you can understand why.